src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+
+    {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {isMarried
+        ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
```

Comment:
Try to avoid nesting ternary operators. Let's create a variable for husband/wife
![image](https://github.com/mate-academy/react_person/assets/96409563/568237fc-2ebb-4522-a226-33a5b6e927af)